layouts/partials/recent_posts.html

```diff
@@ -16,7 +16,7 @@ <h2>{{ .Site.Params.recent_posts.title }}</h2>
 
             <div class="row">
 
-                {{ $posts := .Paginate (where .Data.Pages "Type" "blog") }}
+                {{ $posts := .Paginate (where site.RegularPages "Type" "in" site.Params.mainSections) }}
```

Comment:
I think this should be `.Site.RegularPages` right?

Reply:
`site` is a new global variable introduced in [Hugo 0.53](https://gohugo.io/news/0.53-relnotes/):

> **Global site and hugo var**: Two new global variables in `site` and `hugo`. `hugo` gives you version info etc. (`{{ hugo.Version }}`, `{{ hugo.Environment }}`), but the site is probably more useful, as it allows you to access the current [site’s variables](https://gohugo.io/variables/site/) (e.g. `{{ site.RegularPages }}`) without any context (or “.”).

So `site.RegularPages` and `.Site.RegularPages` should be the same. I don't know when and why to prefer which one, though...

Reply:
> Why mainSection and not blog only ({{ $posts := .Paginate (where .Site.RegularPages "Type" "blog") }}) to have same functionality ?

By using `"blog"`instead of `site.Params.mainSections` you actually hardcode the [content type](https://gohugo.io/content-management/types/) of blog posts to `"blog"`. This is bad for users who want to customize the content type name of their blog posts (e.g. one might prefer `"posts"`, `"articles"` or whatever). 

When using `site.Params.mainSections` the user can simply define what content type(s) should be treated as "articles" and therefore show under recent posts by setting the key `params.mainSections` in `config.toml` (the default should of course be `params.mainSections = "blog"`).

Reply:
ok you convince me, but this PR should update:
- example [config.toml](https://github.com/devcows/hugo-universal-theme/blob/master/exampleSite/config.toml)
- [Readme] (https://github.com/devcows/hugo-universal-theme/blob/master/README.md)

Reply:
Convince too with updated config.toml which wasn't mentioned at the beginning